source/sass/patterns/atoms/image.scss

```diff
@@ -0,0 +1,6 @@
+.image {
+  @include padding($baselinegrid-space-extra_small, block);
+  @include padding(12px, inline);
```

Comment:
No need for 2 includes. Need to remove the magic number too.

Reply:
@davidcmoulton said in an earlier review the need for the 12px.

Reply:
https://github.com/libero/pattern-library/pull/87/files#r280772963? I disagree, we must _never_ include hard-coded values like this, and need to be vigilant. If it can't be anything else, make it a standalone variable.

You might want to check re the design though, as IMO having potentially different inline and block paddings here would be unusual. If that's not meant, then it sounds like the 12px here _can_ be derived from the baseline grid value (even if it's just at the config level).

Reply:
Taking everything into account, this seems like a reasonable compromise:
```
// image.scss

$image-inline-space: $baselinegrid-space-extra_small;

... 

.image {
  @include padding($baselinegrid-space-extra_small, $image-inline-space);
  ...
}
```
It removes the hard coded value, it specifies a distinct variable to define the inline spacing, and it maps it to the baseline grid for now, while at the same time allowing for that to be easily changed should the baseline grid no longer match the inline axis spacing.

Reply:
Hmmm, actually this will likely be needed in more than one file, so rather than put it as a local variable it should be part of config. Continue declaring `$image-inline-space` locally for now and I'll raise a PR to provide it via config. When that's available, the local declaration can be removed.